src/classes/BaseCollector.js

```diff
@@ -6,7 +6,7 @@ class Base extends Emitter {
         this._timeouts = new Set();
         this.filter = filter;
         this.options = options;
-        this.collected = new Map();
+        this.collected = new Collection();

```

Comment:
There is a problem using Map instead of using Collection. Also, the Collection class does not exist in JavaScript. You have to import it first.

Thanks for your interest :3

Reply:
Can you tell me how to pull the content of the collected message from the Map or Collection?

Reply:
you can use `collected.values().next().value`